doc/modules/linear_model.rst

```diff
@@ -903,15 +903,21 @@ in these settings.
 
 .. topic:: **Trade-offs: which estimator?**
 
-   Scikit-learn provides 2 robust regression estimators:
+   Scikit-learn provides 3 robust regression estimators:
    :ref:`RANSAC <ransac_regression>` and
    :ref:`Theil Sen <theil_sen_regression>`
+   :ref:`HuberRegressor <huber_regression>`
 
-   * :ref:`RANSAC <ransac_regression>` is faster, and scales much better
-     with the number of samples
+  * :ref:`HuberRegressor` <huber_regression> should be faster than
```

Comment:
I would reformulate these two sentences. First you say something, then you say it is only sometimes true.
What are the trade-offs? HuberRegressor should be fast with < X samples? When n_samples << n_features?